test/renate/benchmark/models/test_text_transformer.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import torch
+
+from renate.benchmark.models.transformer import HuggingFaceSequenceClassificationTransformer
+
+
+@pytest.mark.parametrize("model_name", ["distilbert-base-uncased", "bert-base-uncased"])
+def test_init(model_name):
+    HuggingFaceSequenceClassificationTransformer(pretrained_model_name=model_name, num_outputs=10)
+
+
+@pytest.mark.parametrize(
+    "model_name,input_dim",
+    [
+        ["distilbert-base-uncased", (128,)],
+        ["bert-base-uncased", (256,)],
+    ],
+)
+def test_text_transformer_fwd(model_name, input_dim):
```

Comment:
if we do not change the `get_backbone` above, can we also test with `prediction_strategy` active?